internal/core/services/autoscaler/policies/roomoccupancy/policy.go

```diff
@@ -85,7 +85,7 @@ func (p *Policy) CalculateDesiredNumberOfRooms(policyParameters autoscaling.Poli
 
 	readyTarget := policyParameters.RoomOccupancy.ReadyTarget
 	if readyTarget >= float64(1) || readyTarget <= 0 {
-		return -1, errors.New("Ready target must be between 0 and 1")
+		return -1, errors.New("Ready target must be between 0 and 0.99")
```

Comment:
why not set the message as `Ready target must be greater than 0 and less than 1`?